### DISCUSSION TITLE
Is the secret sharing links possible while self-hosting Infiscal?

### DISCUSSION THREAD

**Discussion by ScottWilliamAnderson** [Infisical/infisical#2177] [Q&A] [answered]
Hi!

While looking for solutions to securely share secrets & credentials to contributors I work with I came across Infiscal, and was very please to see the secret sharing feature: https://infisical.com/docs/documentation/platform/secret-sharing
Unfortunately, I can't seem to find in the docs if this feature is possible when self-hosting the service on my own network. 

I'm more interested in a secure way to share secrets & credentials, than a way to store & access them, hence my idea to self-host an instance of Infiscal, and share links to it which would let a specific user access what they need. 

I hope this isn't too surface-level a question but, would you know if this is possible?

Many thanks!

**Comment by maidul98** [comment, marked as answer]
Hey @ScottWilliamAnderson , this is possible once you deploy Infisical. You should then be able to go to localhost:80/secret-sharing or in the dashboard if you are logged in
